projects/angular/src/forms/common/STYLES.label.md

```diff
@@ -0,0 +1,48 @@
+# Label
+
+## Variables
+
+| Variable Name                   | Description                            |
+| ------------------------------- | -------------------------------------- |
+| $clr-label-padding-top-bottom   | Padding top and bottom for labels      |
+| $clr-label-padding-left-right   | Padding left and right for labels      |
+| $clr-label-borderwidth          | Border width for labels                |
+| $clr-label-font-color-light     | Font color for light labels            |
+| $clr-label-font-color-dark      | Font color for dark labels             |
+| $clr-label-default-border-color | Default border color for labels        |
+| $clr-label-font-size            | Font size for labels                   |
+| $clr-label-font-weight          | Font weight for labels                 |
+| $clr-label-letter-spacing       | Letter spacing for labels              |
+| $clr-label-bg-hover-color       | Background hover color for labels      |
+| $clr-label-gray-bg-color        | Background color for gray labels       |
+| $clr-label-gray-color           | Font color for gray labels             |
+| $clr-label-purple-bg-color      | Background color for purple labels     |
+| $clr-label-purple-color         | Font color for purple labels           |
+| $clr-label-blue-bg-color        | Background color for blue labels       |
+| $clr-label-blue-color           | Font color for blue labels             |
+| $clr-label-orange-bg-color      | Background color for orange labels     |
+| $clr-label-orange-color         | Font color for orange labels           |
+| $clr-label-light-blue-bg-color  | Background color for light blue labels |
+| $clr-label-light-blue-color     | Font color for light blue labels       |
+| $clr-label-info-bg-color        | Background color for info labels       |
+| $clr-label-info-font-color      | Font color for info labels             |
+| $clr-label-info-border-color    | Border color for info labels           |
+| $clr-label-success-bg-color     | Background color for success labels    |
+| $clr-label-success-font-color   | Font color for success labels          |
+| $clr-label-success-border-color | Border color for success labels        |
+| $clr-label-warning-bg-color     | Background color for warning labels    |
+| $clr-label-warning-font-color   | Font color for warning labels          |
+| $clr-label-warning-border-color | Border color for warning labels        |
+| $clr-label-danger-bg-color      | Background color for danger labels     |
+| $clr-label-danger-font-color    | Font color for danger labels           |
+| $clr-label-danger-border-color  | Border color for danger labels         |
+
+## Classes
+
+| Class Name                      | Description                                    |
+| ------------------------------- | ---------------------------------------------- |
+| .label, a.label                 | Default label class                            |
```

Comment:
Add the smallest possible single selector scope without leading '.'
It should be just "label" here
That's for consistency.
Please apply to the other selectors below.